Running `make setup` on initial repo setup throws error

# Bug Report
## What is the current behavior?
When initially setting up this repo locally, the following error is generated when running the `make setup` command.

```
➜  operationcode_backend git:(master) make setup
docker-compose run web rake db:create
Pulling bundle (operationcodebackend_web:latest)...
ERROR: repository operationcodebackend_web not found: does not exist or no pull access
make: *** [db_create] Error 1
```

## What is the expected behavior?
According to the [Makefile](https://github.com/OperationCode/operationcode_backend/blob/master/Makefile#L42) a successful `db_create db_migrate`

## What steps did you take to get this behavior?
1. Clone the repo; `cd` into it
2. `$ bundle`
3. `$ make setup`

## Additional Info
### Operating System
macOS Sierra 10.12.4

--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@ blog:
 bundle:
 	docker-compose run ${RAILS_CONTAINER} bash -c 'cd /app && bundle'
 
-setup: db_create db_migrate
+setup: build db_create db_migrate
 
 publish: build
 	bin/publish

